client/src/pages/CreateEvent/CreateEvent.js

```diff
@@ -0,0 +1,238 @@
+      //     description : "testB", //propsFormState.description
+      //   }
+      // });
+      
+      const { data } = await addEvent({
+        variables: { 
+          owner: "635c089755ef180f20fb8535", 
+          capacity: 11,//propsFormState.ticket_quantity, 
+          when: propsFormState.date,
+          title: propsFormState.event_title,
+          address: propsFormState.event_street_address,
+          city: propsFormState.event_city,
+          description: propsFormState.description
+         },
+      });
+      console.log(`In createEvent, data is: ${JSON.stringify(data)}`)
+      return data
+
+    } catch (e) {
+      console.error("gql myCustom error, front-end create event")
+      console.error(e);
+    }
+
+  };// END submitFormhelper
+
+   // submit form
+   const handleFormSubmit = async (event) => {
+    event.preventDefault();
+    submitFormhelper(formState)
+  }
+
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <div>
+        <h3>Create Event</h3>
+
+        <form >
+          <label>Event Title</label>
+          <input
+            className="form-input"
+            name="event_title"
+            type="input"
+            id="event_title"
+            onChange={handleChange}
+          />
+          <br />
+          <label>Event Date and Time</label>
+          <br />
+          <br />
+          <TextField
+            id="date"
+            name="date"
+            label="Event Date"
+            type="date"
+            defaultValue=""
+            sx={{ width: 220 }}
+            onChange={handleChange}
+            InputLabelProps={{
+              shrink: true,
+            }}
+          />
+          <br />
+          <br />
+          <TextField
+            id="timeStart"
+            name="timeStart"
+            label="Event Start"
+            type="time"
+            defaultValue="07:30"
+            onChange={handleChange}
+            InputLabelProps={{
+              shrink: true,
+            }}
+            inputProps={{
+              step: 300, // 5 min
+            }}
+            sx={{ width: 150 }}
+          />
+          <br />
+          <br />
+          <label>Event Address</label>
+          <input
+            className="form-input"
+            name="event_street_address"
+            type="input"
+            id="event_street_address"
+            onChange={handleChange}
+          />
+          <label>Event City</label>
+          <input
+            className="form-input"
+            name="event_city"
+            type="input"
+            id="event_city"
+            onChange={handleChange}
+          />
+          <label>Event Postal Code</label>
+          <input
+            className="form-input"
+            name="event_postal_code"
+            type="input"
+            id="event_postal_code"
+            onChange={handleChange}
+          />
+          <br />
+          <label>Ticket Quantity</label>
```

Comment:
Ticket quantity currently provides option to choose negative values, but should have nonnegative values. 
Also, it would be ideal if the sliding option when hovering over the input box can be disabled for better user experience.

Reply:
Resolved by @robyng